docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -436,6 +436,255 @@
     "tab.mean(axis=1)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "fe565b65-fbf2-47ba-a26e-791d09fd4f55",
+   "metadata": {},
+   "source": [
+    "### Table.kurt()\n",
+    "\n",
+    "```\n",
+    "Table.kurt(axis=0, skipna=True, numeric_only=False)\n",
+    "```\n",
+    "\n",
+    "Return unbiased kurtosis over requested axis. Kurtosis obtained using Fisher’s definition of kurtosis (kurtosis of normal == 0.0). Normalized by N-1.\n",
+    "\n",
+    "\r\n",
+    "**Parameters:**\r\n",
```

Comment:
If possible, change all `\r\n` instances in these cells for `\n`. This could be trouble for later if left unchanged and better safe than sorry.

Reply:
Done